### PR TITLE
unittests: rm warning for tests-ipv6_netif

### PR DESCRIPTION
**Pull request by Lotterleben**
I merged https://github.com/RIOT-OS/RIOT/pull/3059 too hastily and missed one warning... This PR should get rid of that one too :)

### CODE REVIEW

File: tests/unittests/tests-ipv6_netif/tests-ipv6_netif.c [hunk 1]
```diff
@@ -95,6 +95,10 @@ static void test_netif_add__success_with_ipv6(void)
     TEST_ASSERT_EQUAL_STRING((char *)exp_addr.u8, (char *)entry->addrs[0].addr.u8);
 }
 
+static void test_netif_add__despite_free_entry(void)
+{
+}
+
```
Comment (authmillenon) [inline tests/unittests/tests-ipv6_netif/tests-ipv6_netif.c]: ?

**Comment by Lotterleben** [comment]
Aaand hasty again. 1 second.

**Comment by Lotterleben** [comment]
fixed & squashed

File: tests/unittests/tests-ipv6_netif/tests-ipv6_netif.c [hunk 2]
```diff
@@ -507,6 +507,7 @@ Test *tests_ipv6_netif_tests(void)
     EMB_UNIT_TESTFIXTURES(fixtures) {
         new_TestFixture(test_ipv6_netif_add__success),
         new_TestFixture(test_netif_add__success_with_ipv6),
+        new_TestFixture(test_netif_add__despite_free_entry),
```
Comment (authmillenon) [inline tests/unittests/tests-ipv6_netif/tests-ipv6_netif.c]: This is not defined.

Comment (authmillenon) [inline tests/unittests/tests-ipv6_netif/tests-ipv6_netif.c]: (still too hasty </treebeardVoice>)

Comment (Lotterleben) [inline tests/unittests/tests-ipv6_netif/tests-ipv6_netif.c]: yes :D I hope I un-hastied everything now.. sorry

**Comment by authmillenon** [comment]
ACK & go, if travis is happy

**Pull request merged**